backend/core/detection.py

```diff
@@ -884,20 +923,22 @@ def commit_hijack(
                     redis_pipeline.sadd("persistent-keys", hijack_value["key"])
                     result = hijack_value
                     self.comm_annotate_hijack(monitor_event, result)
-                    mail_log.info(
-                        "{}".format(
-                            json.dumps(
-                                hijack_log_field_formatter(result),
-                                indent=4,
-                                cls=SetEncoder,
-                            )
-                        ),
-                        extra={
-                            "community_annotation": result.get(
-                                "community_annotation", "NA"
-                            )
-                        },
-                    )
+                    # log in mail only if it is a truly new ongoing hijack
```

Comment:
@slowr what do you think about this? We can also log this as a new hijack and continue (its properties have changed anyway :) ).

Reply:
Actually let's log this but with the outdated_parent field enabled :). Resolving.